src/components/Search.js

```diff
@@ -64,6 +66,13 @@ export const Search = () => {
     })
   }
 
+  const allowInput = () => {
+    if (isOnline === false && consentGranted === false) {
+      return false
+    }
+    return true
```

Comment:
```suggestion
    return isOnline || consentGranted
```

simplify

online : always show the input
offline but has consent : show input
offline but no consent : hide input

Reply:
Love it